docs/user/communication.rst

```diff
@@ -41,99 +45,196 @@ Exemplary exercise detail page with posts:
+    * Post Tag
+* Context filters
+    * Lecture
+    * Exercise
+    * Course-wide topic
+        * Announcement
+        * Organization
+        * Random
+        * Tech Support
+* Other post characteristics
+    * Unresolved Posts
+    * Own (posts current user has authored)
+    * Reacted (posts current user has reacted or replied to)
+* Sorting Configurations (ASC/DESC)
+    * Creation Date
+    * Number of Votes
+    * Number of Answers
 
 A simplified version of this feature is available on the exercise and lecture details page as well.
 
+Verify User Authority
+^^^^^^^^^^^^^^^^^^^^^
+To build trust between users utilizing the system's communication features, we prepend an icon to the author’s name in the
+headers of posts and their replies. The icon we introduce differs according to the role of the author within the course
+and reveals their highest authoritative role. Via tooltips that are shown as users hover over these icons (see images below),
+the system displays a brief explanation of that specific role. This way, the system builds trust in the author, and readers
+can tangibly confirm the origin of the shared information.
+
+Instructor Authority
+
+|instructor-user|
+
+Tutor Authority
+
+|tutor-user|
+
+Student Authority
+
+|student-user|
+
 React on Posts
---------------
+^^^^^^^^^^^^^^
+
+To foster interaction between users, we integrate the well-known emoji reaction bar.
+Each user in the course can react on any post by making use of the emoji selection button.
+The ``+`` emoji serves as the up-voting reaction, which influences the display order of posts.
 
-To foster interaction we integrate the well-known emoji reaction bar.
-Each user in the course can react on every post by making use of the emoji selection button.
-The ``+`` emoji serves as up-voting reaction, which influences the display order of posts.
+Reference Course Entities
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users can reference to different course entities within their posts, such as other posts, course exercises, course lectures,
+and attachments of these lectures. All references are then prepended with icons which are unique to the reference's type,
+to help users distinguish them conveniently. In the image below, we see all possible types of references that can be created
+in an Artemis post.
+|post-with-references|
 
 Reference Posts
----------------
+"""""""""""""""
+
+If users want to refer to other posts, they can integrate a simple pattern including the hashtag (``#``) combined with
+the post identifier. A post's identifier is appended to the post title (such as seen in the screenshots above).
 
-If users want to refer to other posts, they can integrate a simple pattern including the hashtag (``#``) combined with the post identifier.
-A post's identifier is appended to the post title (such as seen in the screenshots above).
+When clicking a post reference used in a post's text, the user is navigated to the referenced post respectfully.
```

Comment:
I think removing `respectfully` makes more sense to me.

Reply:
It has been removed in response to previous comments. Thanks!